<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QWidget" name="centerWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_6" native="true">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(247,247,247);
border-right:1px solid #D6D6D6;</string>
          </property>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>191</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
	background-color: rgb(226,226,226);
	border: none; /* no border for a flat push button */
}
QLineEdit
{
	background-color: rgb(226,226,226);
	border: none; /* no border for a flat push button */
}
</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_addFriend">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>30</y>
             <width>20</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>添加好友</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/tianjia.png); 
	background-color: rgb(226,226,226);
	min-width: 20px;
	max-width: 20px;
	min-height: 20px;
	max-height: 20px;
}
QPushButton::hover {
	background-color: rgb(209,209,209);
	image: url(:/src/tianjia2.png); 
}
QPushButton:pressed {
	image: url(:/src/tianjia.png); 
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget_info">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QListWidget
{
	background-color: rgb(226,224,223);
	border-style: none;
	border-right:1px solid #D6D6D6;
	border-top:1px solid #D6D6D6;
	border-left:1px solid #D6D6D6;
	border-bottom:1px solid #D6D6D6;	
}

QScrollBar:vertical
{
    width:8px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:9px;
    padding-bottom:9px;
}
QScrollBar::handle:vertical
{
    width:8px;
    background:rgba(0,0,0,25%);
    border-radius:4px;
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:8px;
    background:rgba(0,0,0,50%);
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical
{
    height:9px;width:8px;
    border-image:url(:/images/a/3.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical
{
    height:9px;width:8px;
    border-image:url(:/images/a/1.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
    height:9px;width:8px;
    border-image:url(:/images/a/4.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
    height:9px;width:8px;
    border-image:url(:/images/a/2.png);
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(245,245,245);</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,0,0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushBtn_hide">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/min.png); /* 最小化 */
	min-width: 30px;
	max-width: 30px;
	min-height: 25px;
	max-height: 25px;
}
QPushButton::hover {
	background-color: rgba(166,166,166, 50); 
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushBtn_max">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/max.png); /* 最小化 */
	min-width: 30px;
	max-width: 30px;
	min-height: 25px;
	max-height: 25px;
}
QPushButton::hover {
	background-color: rgba(166,166,166, 50); 
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QPushButton" name="pushBtn_close">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/close.png); /* 最小化 */
	min-width: 30px;
	max-width: 30px;
	min-height: 25px;
	max-height: 25px;
}
QPushButton::hover {
	background-color: #f57575; 
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_7" native="true"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_7" columnstretch="0,0,1,0,0">
             <item row="0" column="1">
              <widget class="QLabel" name="label_info">
               <property name="font">
                <font>
                 <family>新宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>聊天信息</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QPushButton" name="pushButton">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/more.png); 
	min-width: 30px;
	max-width: 30px;
	min-height: 20px;
	max-height: 20px;
}
QPushButton::hover {
	image: url(:/src/more2.png); 
}
QPushButton:pressed {
	image: url(:/src/more.png); 
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QWidget" name="widget_8" native="true"/>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_15" native="true">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QWidget" name="widget_16" native="true">
               <property name="minimumSize">
                <size>
                 <width>10</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QStackedWidget" name="stackedWidget_Msg">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page"/>
             <widget class="QWidget" name="page_2"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QWidget" name="widget_4" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_11" columnstretch="0,0,0,0,0,1">
             <property name="horizontalSpacing">
              <number>12</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="3">
              <widget class="QPushButton" name="pushButton_screenshot">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/screenshot.png); 
	min-width: 20px;
	max-width: 20px;
	min-height: 20px;
	max-height: 20px;
}
QPushButton::hover {
	image: url(:/src/screenshot2.png); 
}
QPushButton:pressed {
	image: url(:/src/screenshot.png); 
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QWidget" name="widget_11" native="true"/>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButton_file">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/file.png); 
	min-width: 20px;
	max-width: 20px;
	min-height: 20px;
	max-height: 20px;
}
QPushButton::hover {
	image: url(:/src/file2.png); 
}
QPushButton:pressed {
	image: url(:/src/file.png); 
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_emoj">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/emoj.png); 
	min-width: 20px;
	max-width: 20px;
	min-height: 20px;
	max-height: 20px;
}
QPushButton::hover {
	image: url(:/src/emoj2.png); 
}
QPushButton:pressed {
	image: url(:/src/emoj.png); 
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QPushButton" name="pushButton_chat">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/chat.png); 
	min-width: 20px;
	max-width: 20px;
	min-height: 20px;
	max-height: 20px;
}
QPushButton::hover {
	image: url(:/src/chat2.png); 
}
QPushButton:pressed {
	image: url(:/src/chat.png); 
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_12" native="true">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QWidget" name="widget_13" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_13">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_14" native="true">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QTextEdit" name="textEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>65</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>65</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>新宋体</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
border: none; /* no border for a flat push button */
</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_9" columnstretch="1,0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushBtn_send">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(96,96,96);
	border-radius: 5px;	
	border-style: solid;
	background-color: rgb(245,245,245);

	border-right:1px solid #E5E5E5;
	border-top:1px solid #E5E5E5;
	border-left:1px solid #E5E5E5;
	border-bottom:1px solid #E5E5E5;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(18,150,17);
}

QPushButton:pressed {
	background-color: rgb(7,193,96);
}</string>
               </property>
               <property name="text">
                <string>发送(S)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_9" native="true"/>
             </item>
             <item row="0" column="2">
              <widget class="QWidget" name="widget_10" native="true">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(46,46,46);</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
    background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.70 rgba(0, 0, 0, 0), stop:0.80 rgba(136, 194, 255, 100), stop:1 rgba(0, 0, 0, 0));
}
QLabel:hover{
    background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.70 rgba(0, 0, 0, 0), stop:0.80 rgba(186, 224, 255, 100), stop:1 rgba(0, 0, 0, 0));
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="src.qrc">:/src/头像.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_emoj_2">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>110</y>
           <width>25</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

	border: none; /* no border for a flat push button */
	image: url(:/src/wechat.png); 
	min-width: 25px;
	max-width: 25px;
	min-height: 25px;
	max-height: 25px;
}
QPushButton::hover {
	image: url(:/src/wechat3.png); 
}
QPushButton:checked{
	image: url(:/src/wechat2.png); 
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
